stories/notices.stories.js

```diff
@@ -95,4 +97,12 @@ storiesOf( 'Global/Notices', module )
 			LearnMore={ LearnMore }
 			notice="This is a suggestion."
 		/>
-	) );
+	) )
+	.add( 'ErrorNotice', () => {
```

Comment:
@sashadoes This is the old way of writing stories which we moved away from. Instead can you create `assets/js/components/ErrorNotice.stories.js` and add the story in there. Same for `ErrorText`. You can use other `stories.js` as reference.